### DISCUSSION TITLE
Welcome to on-my-way Discussions!

### DISCUSSION THREAD

**Discussion by masterivanic** [masterivanic/on-my-way#1] [Announcements]
## 👋 Hey, Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Add content to this repo to help other.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by IT-WIBRC** [comment]
Yo @masterivanic I think it will also be good to add a section where to share the links about places where we can have free or paid certification

**Reply by masterivanic** [reply]
Nice, i  add only platform to get certify, i will add a section about.

**Comment by IT-WIBRC** [comment]
Also, introduce at the bottom the instructions on what to do if we want to contribute to this project. `Eg.` 1. Fork the project, 2...

**Reply by masterivanic** [reply]
Yes indeed, i will write a memo and rules to everyone who want to contribute 
Feel free to propose it and open a pull request :)

**Reply by IT-WIBRC** [reply]
Ok i will
